src/test/scala-2/chiselTests/LayerSpec.scala

```diff
@@ -216,6 +216,23 @@ class LayerSpec extends AnyFlatSpec with Matchers with FileCheck {
 
   }
 
+  they should ", when enabled, not propagate to child modules" in {
+
+    class Bar extends RawModule
+
+    class Foo extends RawModule {
+      layer.enable(A)
+      val bar = Module(new Bar)
+    }
+
+    ChiselStage.emitCHIRRTL(new Foo, Array("--full-stacktrace")).fileCheck() {
```

Comment:
```suggestion
    ChiselStage.emitCHIRRTL(new Foo).fileCheck() {
```
Unnecessary (but benign) argument.

Reply:
Thanks for catching this! I don't like when these stay in the tests. Fixed in: https://github.com/chipsalliance/chisel/pull/4840/commits/607aa6daf6a89764463f08933e9665956e8ea182